docs/sdks/server-sdks/go.md

```diff
@@ -40,13 +40,13 @@ func main() {
 	assignmentLogger := NewExampleAssignmentLogger()
 
 	eppoClient = eppoclient.InitClient(eppoclient.Config{
-		ApiKey:           "<your_api_key>",
+		ApiKey:           "<your_sdk_key>",
 		AssignmentLogger: assignmentLogger,
 	})
 }
 ```
 
-After initialization, the SDK begins polling Eppo’s API at regular intervals to retrieve the most recent experiment configurations such as variation values and traffic allocation. The SDK stores these configurations in memory so that assignments thereafter are effectively instant. For more information, see the [architecture overview](/sdks/overview) page.
+After initialization, the SDK begins polling Eppo’s  at regular intervals to retrieve the most recent experiment configurations such as variation values and traffic allocation. The SDK stores these configurations in memory so that assignments thereafter are effectively instant. For more information, see the [architecture overview](/sdks/overview) page.
```

Comment:
```suggestion
After initialization, the SDK begins polling Eppo's API at regular intervals to retrieve the most recent experiment configurations such as variation values and traffic allocation. The SDK stores these configurations in memory so that assignments thereafter are effectively instant. For more information, see the [architecture overview](/sdks/overview) page.
```